SOLUTION OF QUIZ/q1.py

```diff
@@ -0,0 +1,34 @@
+### 8.  Create Contact.csv file which contain name,contact. Use "|" as Delimiter.
+### Search particular contact by entering person name . 
+
+import csv
+
+with open("C://22BCA206//python//contact_file1.csv",'w',newline='') as fileobj:
+    obj=csv.writer(fileobj)
+    field=['NAME','CONTACT NUMBER']
+    obj.writerow(field)
+    # 5 records insert directly
+    records=[['Jay',9923456786],['Shree',9923356796],['Ram',6323456786],['Shiv',7923456786],['Krishna',6683456786]]
+    obj.writerows(records)
+
+with open("C://22BCA206//python//contact_file.csv",'r',newline='') as ob:
+    data=csv.reader(ob)
+    for i in data:
+        print("{:^10} {:^10}".format(i[0],i[1]))

```

Comment:
please tell me the meaning of this statement...

Reply:
In this line,i use .format( )
{:^10} :-  i use this for print data or record in proper format.
^ :-  this indicates align center.